core-skills/c-sharp-and-dotnet/README.md

```diff
@@ -0,0 +1,84 @@
+# C# and .NET Core
+
+## Who is this skill for?
+
+This skill is intended for developers who have already completed the TDD core skill, and who have not worked with .NET previously (or want to brush up) and are likely to join a delivery team that is working in C#.
+Through this course, we'll be making some assumptions that you're already familiar with at least one programming language (static typed or dynamic).
+
+## Summary
+### Mantis Shrimp
+1. Can set up a working project in .NET using an IDE
+2. Can install and manage dependencies for the project
+3. Can create and run unit tests using a test framework
+4. Understands basic concepts of the C# language and ecosystem
+
+### Goat
+1. Understands some deeper object oriented programming concepts
+2. Able to demonstrate common design patterns implemented using C# language features
+3. Understands how to build API functionality using common frameworks (MVC, EF)
+4. Understands how to use the dotnet command line utilities to perform common tasks
+5. Can effectively debug using the interactive debugger (IDE)
+
+### Eagle
+1. Demonstrate an understanding of functional programming in C#
+2. Understanding of the .NET virtual machine
+3. Understanding the impact of choices on performance
+4. Demonstrate ability to combine understanding of the above into a working API in C# with .NET
+
+## Resources
+Coming soon
+
+## Mark Scheme
+### Mantis Shrimp
+
+- Can explain the difference between .NET, .NET Core and Framework
+- Can explain what the .NET Standard is 
+- Can explain the difference between C#, Visual Basic and F#
```

Comment:
Is this really a C# skill?

Reply:
Debatable - but you _will_ come across documentation in both C# and VB, or come across code snippets that are unfamiliar. Also, F# features often make their way into C# in later updates, so knowing about what exists in the MS ecosystem is relatively important